test/e2e/storage/drivers/in_tree.go

```diff
@@ -1857,10 +1857,10 @@ func (l *localDriver) PrepareTest(f *framework.Framework) (*testsuites.PerTestCo
 	}
 
 	return &testsuites.PerTestConfig{
-			Driver:         l,
-			Prefix:         "local",
-			Framework:      f,
-			ClientNodeName: l.node.Name,
+			Driver:             l,
+			Prefix:             "local",
+			Framework:          f,
+			ClientNodeSelector: map[string]string{v1.LabelHostname: l.node.Name},
```

Comment:
local volumes also support topology. Need to investigate if this is still needed.